.github/workflows/samples-dotnet-standard.yaml

```diff
@@ -15,8 +27,13 @@ jobs:
       fail-fast: false
       matrix:
         sample:
-          # clients
-          - samples/client/petstore/csharp/OpenAPIClient-generichost-netstandard2.0
+          # build C# API client (netstandard)
+          - samples/client/petstore/csharp/OpenAPIClient/**
+          - samples/client/petstore/csharp/*netstandard*/**
+          # build C# API client (netstandard with ConditionalSerialization)
+          - samples/client/petstore/csharp/OpenAPIClient-ConditionalSerialization/**
+          # build C# API client (netstandard httpclient)
+          - samples/client/petstore/csharp/OpenAPIClient-httpclient/**
```

Comment:
for `sample` under matrix, I thought we must use the full path without using **. Am I right?

Reply:
Oh, I've just copied from top.
Done.